modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -1,48 +1,27 @@
 = Create a Serverless Cluster
 :description: Learn how to create a Serverless cluster.
 :page-cloud: true
-:page-beta: true
 
-Redpanda Serverless is the fastest and easiest way to start event streaming in the cloud. 
+To start using Redpanda Serverless, sign up for a free trial. If you want to continue after the trial ends, you can enter a credit card and pay as you go. 
```

Comment:
Maybe we can highlight that no credit card is required to start the trial? Customers usually love this sentence.